### DISCUSSION TITLE
v2.3.0 - Released and available in the App Store

### DISCUSSION THREAD

**Discussion by afterxleep** [afterxleep/Bouncer#48] [Announcements]
## What's Changed

* iOS 16 message categories support.  Sort your messages into more categories
* Add support for Importing/Export Block Lists by @martingordon
* Improved filter matching
* Case-sensitive filtering
* Add Persian localization by @sdtaheri
* Added French and German support

## New Contributors (Thank you!)
* @sdtaheri 
* @martingordon made their first contribution in https://github.com/afterxleep/Bouncer/pull/45

**Full Changelog**: https://github.com/afterxleep/Bouncer/compare/v2.2.2...v2.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/afterxleep/Bouncer/releases/tag/v2.3.0'>v2.3.0</a>.</em>

**Comment by sdtaheri** [comment]
Good job @afterxleep! I didn't you you were gonna release it. There are lots of untraslated texts for `fa` in iOS 16 categories. Will send a pull request soon.

**Reply by afterxleep** [reply]
👋 I did update the localized strings and made translations using a software solution.  Please make sure you check out the latest version.
